netdata.spec.in

```diff
@@ -1,90 +1,100 @@
+#
+# Conditional build:
+%bcond_without	systemd		# systemd
+%bcond_without	nfacct		# build with nfacct plugin
+
 %if 0%{?fedora} || 0%{?rhel} >= 7
-%global do_systemd 1
 %else
-%global do_systemd 0
+%undefine	with_systemd
 %endif
 
 Summary:	Linux real time system monitoring, over the web
 Name:		@PACKAGE_NAME@
 Version:	@PACKAGE_RPM_VERSION@
 Release:	@PACKAGE_RPM_RELEASE@%{?release_suffix}%{?dist}
-License:	GPLv2+
-URL:		http://firehol.org
-Source:		%{name}-@PACKAGE_VERSION@.tar.bz2
-
+License:	GPL v3+
+Group:		Applications/System
+Source0:	https://github.com/firehol/netdata/archive/v%{version}/%{name}-%{version}.tar.gz
+URL:		http://netdata.firehol.org/
+BuildRequires:	autoconf
+BuildRequires:	automake
 BuildRequires:	libmnl-devel
+%{?with_nfacct:BuildRequires:	libnetfilter_acct-devel}
+BuildRequires:	pkgconfig
 BuildRequires:	zlib-devel
-Requires:	libmnl
-Requires:	zlib
-%if !0%{?rhel}
-BuildRequires:	libnetfilter_acct-devel
-Requires:	libnetfilter_acct
-%endif
-%if 0%{do_systemd}
-BuildRequires:		systemd
-Requires(post):		systemd
+%if %{with systemd}
+BuildRequires:	systemd
+Requires(post):	systemd
 Requires(preun):	systemd
 Requires(postun):	systemd
 %endif
-
-BuildRoot:	%{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+BuildRoot:	%{tmpdir}/%{name}-%{version}-root-%(id -u -n)
 
 %description
-Real-time performance monitoring, in the greatest possible detail!
+netdata is the fastest way to visualize metrics. It is a resource
+efficient, highly optimized system for collecting and visualizing any
+type of realtime timeseries data, from CPU usage, disk activity, SQL
+queries, API calls, web site visitors, etc.
+
+netdata tries to visualize the truth of now, in its greatest detail,
+so that you can get insights of what is happening now and what just
+happened, on your systems and applications.
+
+%prep
+%setup -q
+
+%build
+%{__aclocal} -I m4
+%{__autoconf}
+%{__autoheader}
+%{__automake}
+%configure \
+	--docdir=%{_docdir}/%{name}-%{version} \
+	--with-zlib \
+	--with-math \
+	%{?with_nfacct:--enable-plugin-nfacct} \
+	--with-user=netdata
+%{__make} %{?_smp_mflags}
+
+%install
+rm -rf $RPM_BUILD_ROOT
+%{__make} %{?_smp_mflags} install \
+	DESTDIR=$RPM_BUILD_ROOT
+
+%{__rm} $RPM_BUILD_ROOT/var/{cache,log}/netdata/.keep
```

Comment:
i initially moved the .keep removal to %prep, but the build system relied on the .keep files